applications/fake-services/app/templates/rqlite-ui-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rqliteui 
```

Comment:
I'd also consider templating the names of services/deployments etc so that you future proof for being able to support multiple installs into a single namespace. Just a general best practice to make sure the `releaseName` is included. 

Will leave it up to you, if you want to address that now or in a follow up story.